docs/core/entity.md

```diff
@@ -461,6 +461,10 @@ State attributes which are not suitable for state history recording should be ex
 - `_entity_component_unrecorded_attributes: frozenset[str]` may be set in a base component class, e.g. in `light.LightEntity`
 - `_unrecorded_attributes: frozenset[str]` may be set in an integration's platform e.g. in an entity class defined in platform `hue.light`.
 
+The use of the `MATCH_ALL` constant can be used to exclude all attributes instead of typing them separately. This can be useful for integrations providing attributes which is not known or when you simply want to exclude all without typing them separately.
```

Comment:
```suggestion
The `MATCH_ALL` constant can be used to exclude all attributes instead of typing them separately. This can be useful for integrations providing unknown attributes or when you simply want to exclude all without typing them separately.
```